### ISSUE TITLE
Data manipulation should apply to validation set

### ISSUE THREAD

**Issue by lannelin** [alan-turing-institute/arc-model-similarities-phase-2#23]
Currently dataset manipulation only applies to training set and validation and test are kept as original.

As validation data may be used by the attacker in model development, it should follow the same process as training data. 
I.e. if the attacker only has a blurred set of images available to them, this should be the case for both the training and validation data.

**Comment by lannelin** [comment]
closed by #27
